modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3384,6 +3384,168 @@ Response::
 | String ID of the query.
 |===
 
+
+=== Get information about a template query
+
+`GET /gsql/library?<function name>=pkg1.pkg2.<template name>`
```

Comment:
why is **function name** surrounded by angle brackets?

Reply:
I misunderstood what it was doing, fixed.